lib/jekyll/jekyll-sitemap.rb

```diff
@@ -8,20 +8,20 @@ class JekyllSitemap < Jekyll::Generator
     # Main plugin action, called by Jekyll-core
     def generate(site)
       @site = site
-      unless sitemap_exists?
-        write
-        @site.keep_files ||= []
-        @site.keep_files << "sitemap.xml"
-      end
+      @site.pages << sitemap_content unless sitemap_exists?
     end
 
+    private
```

Comment:
You are correct, but since we have not yet reached `1.0.0`, a minor bump will suffice.

From [semver.org](http://semver.org/#spec-item-5):
> Version 1.0.0 defines the public API. The way in which the version number is incremented after this release is dependent on this public API and how it changes.

So, technically, we have not yet defined a public API.